ms-engine/src/main/java/net/trajano/ms/engine/internal/spring/CdiScopeMetadataResolver.java

```diff
@@ -0,0 +1,39 @@
+package net.trajano.ms.engine.internal.spring;
+
+import java.util.Set;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.RequestScoped;
+
+import org.springframework.beans.factory.annotation.AnnotatedBeanDefinition;
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.context.annotation.AnnotationScopeMetadataResolver;
+import org.springframework.context.annotation.ScopeMetadata;
+import org.springframework.context.annotation.ScopedProxyMode;
+
+public class CdiScopeMetadataResolver extends AnnotationScopeMetadataResolver {
+
+    @Override
+    public ScopeMetadata resolveScopeMetadata(final BeanDefinition definition) {
+
+        if (definition instanceof AnnotatedBeanDefinition) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)